free-podcasts-screencasts-en.md

```diff
@@ -71,6 +71,7 @@
 * [Data Stories](http://datastori.es) (podcast)
 * [Learning Machines 101](http://www.learningmachines101.com) (podcast)
 * [Linear Digressions](http://www.lineardigressions.com) (podcast)
+* [Not So Standard Deviations](https://nssdeviations.com/) (podcast)
```

Comment:
Hello,

Great resource! The reason the build is failing is due to the trailing slash "/"

```suggestion
* [Not So Standard Deviations](https://nssdeviations.com) (podcast)
```

Check out [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) to see that.

> on root domains, strip the trailing slash: http://example.com instead of http://example.com/

Fixing this issue would allow the build to pass and get your PR merged!